packages/extension-placeholder/src/placeholder.ts

```diff
@@ -63,6 +63,15 @@ export interface PlaceholderOptions {
    * @default false
    */
   includeChildren: boolean
+  
+  /**
+   * **Controls if the placeholder should be shown for atom nodes.**
+   *
+   * If true, the placeholder will be shown for atom nodes.
+   * If false, the placeholder will not be shown for atom nodes.
+   * @default false
+   */
+  showWhenAtom: boolean;
```

Comment:
I disagree with making this an option, people should not have to care. You can also just disable the placeholder with css so I don't see why we would need to even modify the JS here